Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -226,13 +246,78 @@ public final class SecureEnclaveValet: NSObject {
 
     // MARK: Internal Properties
 
+    /// A provider of an `LAContext` used when needing to manually evaluate an authentication policy
+    /// prior to retrieving a value from the keychain in order to allow the evaluation prompt to show a
+    /// custom fallback button to the user.  Should always return a new instance of `LAContext` in
+    /// practice, but specified as an internal var to allow it to be overridden in tests.
```

Comment:
Likely out of scope for this PR, but the fact that you had to make this property `internal` has me thinking that maybe we should make our initializers `internal` to enable properties like these to be made `private`.

Reply:
Agreed, and would rather inject this than override it. I can followup after this PR with that change.

Reply:
Also means that this can be immutable.